src/components/common/Problem.vue

```diff
@@ -24,49 +37,105 @@ export default defineComponent({
 			default: ''
 		}
 	},
+	directives: {
+		resize: {
+			mounted(el: IFrameComponent, { value = {} }) {
+				iframeResizer(value, el);
+			},
+			beforeUnmount(el: ExtendedIFrameComponent) {
+				el.iFrameResizer.removeListeners();
+			}
+		}
+	},
 	setup(props){
-		const html: Ref<string> = ref('');
-		const _file: Ref<string> = ref(props.file);
-		const renderDiv = ref<HTMLElement>();
+		const html = ref('');
+		const _file = ref(props.file);
+		const problemIframe = ref<HTMLIFrameElement>();
+		const isLoading = ref(true);
 
 		async function loadProblem() {
-			const formData = new FormData();
-			formData.set('problemSeed', '1');
-			formData.set('sourceFilePath', _file.value);
-			formData.set('outputFormat', 'static');
-
-			let value;
-			try {
-				const response = await axios.post('/renderer/render-api', formData,
-					{ headers: { 'Content-Type': 'multipart/form-data' } });
-
-				value = (response.data as { renderedHTML: string }).renderedHTML;
-			} catch(e) {
-				return;
+			isLoading.value = true;
+			html.value = '';
+
+			// these overrides should be handled by passing requests
+			// through the backend API, or by using JWE
+			const overrides = {
+				'problemSeed': '1',
+				'sourceFilePath': _file.value,
+				'outputFormat': 'classic'
+			};
+			const value = await fetchProblem(new FormData(), RENDER_URL, overrides);
+
+			html.value = value;
+			console.log('HTML updated...');
+
+			return Promise.resolve()
+				.then(() => {
+					/* eslint-disable @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call */
+					window.MathJax.typesetPromise();
+					/* eslint-enable @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call */
+				});
+			// .catch();  // need to figure out a more robust way of handling an error like this.
+		}
+
+		async function submitHandler(form: HTMLFormElement, btn: HTMLButtonElement) {
+			const submitUrl = form.getAttribute('action') ?? '/renderer/render-api';
+			// submitAction is a global function from renderer - prepares for submit
+			const submitAction = (window as Window).submitAction;
+       		if(typeof submitAction === 'function') submitAction();
```

Comment:
This is what I saw that made me wonder if you knew that the global submitAction of webwork no longer exists. These lines will not work and are also not needed.  When the form is submitted the event listeners attached in ww_applet_support.js will run.